{"hierarchy":{"paths":[["doc:\/\/BlueConnect\/documentation\/BlueConnect","doc:\/\/BlueConnect\/documentation\/BlueConnect\/BleCentralManagerProxyError"]]},"primaryContentSections":[{"kind":"declarations","declarations":[{"languages":["swift"],"tokens":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"text":"connectionTimeout","kind":"identifier"}],"platforms":["macOS"]}]},{"kind":"content","content":[{"anchor":"discussion","level":2,"type":"heading","text":"Discussion"},{"type":"paragraph","inlineContent":[{"text":"This error is thrown when a peripheral connection attempt exceeds the allowed time limit.","type":"text"}]}]}],"sections":[],"abstract":[{"type":"text","text":"Indicates that a timeout occurred during peripheral connection."}],"variants":[{"paths":["\/documentation\/blueconnect\/blecentralmanagerproxyerror\/connectiontimeout"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/BlueConnect\/documentation\/BlueConnect\/BleCentralManagerProxyError\/connectionTimeout","interfaceLanguage":"swift"},"schemaVersion":{"minor":3,"patch":0,"major":0},"metadata":{"externalID":"s:11BlueConnect27BleCentralManagerProxyErrorO17connectionTimeoutyA2CmF","roleHeading":"Case","title":"BleCentralManagerProxyError.connectionTimeout","modules":[{"name":"BlueConnect"}],"role":"symbol","fragments":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"text":"connectionTimeout","kind":"identifier"}],"symbolKind":"case"},"kind":"symbol","references":{"doc://BlueConnect/documentation/BlueConnect/BleCentralManagerProxyError":{"kind":"symbol","role":"symbol","navigatorTitle":[{"text":"BleCentralManagerProxyError","kind":"identifier"}],"abstract":[{"text":"An enumeration representing various errors that can occur in the ","type":"text"},{"code":"BleCentralManagerProxy","type":"codeVoice"},{"text":".","type":"text"}],"type":"topic","title":"BleCentralManagerProxyError","identifier":"doc:\/\/BlueConnect\/documentation\/BlueConnect\/BleCentralManagerProxyError","url":"\/documentation\/blueconnect\/blecentralmanagerproxyerror","fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"text":"BleCentralManagerProxyError","kind":"identifier"}]},"doc://BlueConnect/documentation/BlueConnect":{"role":"collection","url":"\/documentation\/blueconnect","title":"BlueConnect","identifier":"doc:\/\/BlueConnect\/documentation\/BlueConnect","kind":"symbol","abstract":[],"type":"topic"},"doc://BlueConnect/documentation/BlueConnect/BleCentralManagerProxyError/connectionTimeout":{"type":"topic","title":"BleCentralManagerProxyError.connectionTimeout","abstract":[{"type":"text","text":"Indicates that a timeout occurred during peripheral connection."}],"role":"symbol","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"text":"connectionTimeout","kind":"identifier"}],"identifier":"doc:\/\/BlueConnect\/documentation\/BlueConnect\/BleCentralManagerProxyError\/connectionTimeout","kind":"symbol","url":"\/documentation\/blueconnect\/blecentralmanagerproxyerror\/connectiontimeout"}}}